Index: app/src/main/java/com/example/kelompok2/Activities/SettingsActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/kelompok2/Activities/SettingsActivity.kt b/app/src/main/java/com/example/kelompok2/Activities/SettingsActivity.kt
--- a/app/src/main/java/com/example/kelompok2/Activities/SettingsActivity.kt	
+++ b/app/src/main/java/com/example/kelompok2/Activities/SettingsActivity.kt	(date 1729828809693)
@@ -11,11 +11,12 @@
 import androidx.appcompat.app.AppCompatActivity
 import com.bumptech.glide.Glide
 import com.example.kelompok2.R
-import com.github.dhaval2404.imagepicker.ImagePicker
+import com.google.android.gms.cast.framework.media.ImagePicker
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.storage.FirebaseStorage
 
+
 class SettingsActivity : AppCompatActivity() {
 
     private lateinit var ivProfileImage: ImageView
